test/src/test/testtool/ManagedFileHandler/scenario01.properties

```diff
@@ -0,0 +1,18 @@
+scenario.description = MFH process one xml file
+
+include = common.properties
+
+dir.init.filename						= scenario01/dir-init.xml
+
+step1.jdbc.deleteRecords.read			= common/rowsupdated-dummy.xml
+
+step2.dir.init.write					= common/dummy.txt
```

Comment:
you can also use `writeLine` that way you don't have to create a new (empty) file